### PR TITLE
eus-upgrades-mvp: Fix MCO constraints examples

### PR DESCRIPTION
**Pull request by sdodson**

### CODE REVIEW

File: enhancements/update/eus-upgrades-mvp.md [hunk 1]
```diff
@@ -122,19 +122,24 @@ version away and *block* upgrades for removals which are imminent.
...
 ```yaml
 node.status.nodeInfo.kubeletVersion:
 - v1.20
-- v1.19
 ```
 
-If the policy were to change to allow for a version skew of N-2, v1.18 would be
+If the policy were to change to allow for a version skew of N-2, v1.19 would be
 added to the list of acceptable matches. The MCO is not responsible for defining
 these constrains and constraints are only widened whenever we have CI testing
```
Comment (dhellmann) [inline enhancements/update/eus-upgrades-mvp.md:134]: While we're in here...

```suggestion
these constraints and constraints are only widened whenever we have CI testing
```

**Review by dhellmann — APPROVED** [review]
This update looks right. I noticed one typo near a change you were already making so proposed an update.

File: enhancements/update/eus-upgrades-mvp.md (hunk 1 above)
Comment (sdodson) [reply enhancements/update/eus-upgrades-mvp.md]: Thanks! I gave this section an edit after reading over it again and I've addressed both of those.

**Comment by sdodson** [comment]
@darkmuggle PTAL, I just noticed this error which may have led to some confusion for you and MCO team.

**Comment by kikisdeliveryservice** [comment]
would like this discuss this in our meeting today :smile:

**Review by wking — APPROVED** [review]
/lgtm

This makes sense to me and adds clarity to Scott's plan.  Any follow-up pivots can come in via follow-up PRs.

**Pull request merged**